plugins/follow_me/follow_me.h

```diff
@@ -0,0 +1,170 @@
+        };
+        // Default Configuration
+        static const float DEF_HEIGHT_M; /**< @brief Default height above home: 8.0 mts */
+        static const float DEF_FOLLOW_DIST_M; /**< @brief Default Follow distance: 8.0 mts */
+        static const FollowDirection DEF_FOLLOW_DIR; /**< @brief Default follow direction: BEHIND */
+        static const float DEF_RESPONSIVENSS; /**< @brief Default responsiveness to target: 0.5 */
+
+        // Min & Max ranges as per PX4 FollowMe Configuration.
+        static const float MIN_HEIGHT_M; /**< @brief Min follow height, in mts. >= 8.0m */
+        static const float MIN_FOLLOW_DIST_M;  /**< @brief Min follow distance from, in mts. >= 1.0m */
+        static const float MIN_RESPONSIVENESS; /**< @brief Min responsiveness: 0.0 */
+        static const float MAX_RESPONSIVENESS; /**< @brief Max responsiveness: 1.0 */
+
+        float min_height_m = DEF_HEIGHT_M; /**< @brief Min follow height above home */
+        float follow_dist_m =
+            DEF_FOLLOW_DIST_M; /**< @brief Follow horizontal distance b/w Vehicle & GCS. */
+        FollowDirection follow_dir = DEF_FOLLOW_DIR; /**< @brief Side to follow target from. */
+        float responsiveness = DEF_RESPONSIVENSS; /**< @brief Responsiveness: Range (0.0 to 1.0)*/
+
+        /**
+         * @brief Human-readable string of enum `FollowDirection`.
+         * @param dir follow direction
+         * @return std::string representing given direction
+         */
+        static std::string to_str(FollowDirection dir);
+    };
+
+    /**
+     * @brief Gets current FollowMe configuration.
+     * @return current FollowMe configuration.
+     * @sa set_config(const Config &)
+     */
+    const Config &get_config() const;
+
+    /**
+     * @brief Applies FollowMe configuration by sending them to device.
+     * @param config FollowMe configuration to be applied.
+     * @return true if configuration is applied successfully, false if config values are out-of-range.
+     *         In case of failure, last configuration is preserved.
+     * @sa get_config()
+     */
+    bool set_config(const Config &config);
+
+    /**
+     * @brief Checks whether FollowMe is active.
+     * @return true if FollowMe is active, false otherwise.
+     */
+    bool is_active() const;
+
+    /**
+     * @brief Represents geographic location of the target in motion
+     */
+    struct TargetLocation {
+        double lattitude_deg; /**< @brief Latitude */
+        double longitude_deg; /**< @brief Longitude */
+        double absolute_altitude_m; /**< @brief AMSL, in meters */
+
+        float velocity_x_mps; /**< @brief X-velocity in m/s2 */
+        float velocity_y_mps; /**< @brief Y-velocity in m/s2 */
+        float velocity_z_mps; /**< @brief Z-velocity in m/s2 */
+    };
+
+    /**
+     * @brief Sets current location of the moving target
+     * @note App can obtain location of the moving target from Location framework of the underlying platform. <br>
```

Comment:
Remove all the html br line breaks. If you really need a para break, add space between lines.